src/components/views/dialogs/spotlight/SpotlightDialog.tsx

```diff
@@ -510,6 +510,7 @@ const SpotlightDialog: React.FC<IProps> = ({ initialText = "", initialFilter = n
             metricsViaKeyboard: viaKeyboard,
             room_id: room.roomId,
             room_alias: room.roomAlias,
+            auto_join: room.autoJoin,
```

Comment:
might be worth to also pass `peek: false` in this case to save it possibly trying to peek